operations/app/src/environments/tf

```diff
@@ -19,6 +20,14 @@ SCRIPT_DIR="$(dirname "$(realpath "${0}")")"
 CONFIGS_DIR="${SCRIPT_DIR?}/configurations"
 PLAN_DIR="${SCRIPT_DIR?}/../../plan"
 
+# IP and services that need to be whitelisted in the firewall
+PUBLIC_IP="$(curl -s ifconfig.me)"
+PUBLIC_IP_CIDR="${PUBLIC_IP?}/32"
+PUBLIC_IP_HASH="$(echo -n "${PUBLIC_IP_CIDR?}" | sha1sum | awk '{print $1}')"
+declare -a KeyVaults=( "-keyvault" "-appconfig" "-clientconfig" )
+declare -a StorageAccounts=( "storageaccount" "partner" "candidate" "candpartner" )
+declare -A FunctionApps=( ["-functionapp"]="production,candidate" )
```

Comment:
This funky syntax is a KVP with the value being an array of slots that exist in the function app. Each slot has its own independent firewall rules.